internal/provider/resource_aci_relation_to_domain_test.go

```diff
@@ -370,13 +339,16 @@ resource "aci_relation_to_domain" "test" {
 `
 
 const testConfigFvRsDomAttAllDependencyWithFvAEPg = testDependencyConfigFvRsDomAtt + testConfigFvAEPgMinDependencyWithFvAp + `
+data "aci_system" "version" {
+  pod_id    = "1"
+  system_id = "1"
+}
+
 resource "aci_relation_to_domain" "test" {
   parent_dn = aci_application_epg.test.id
   target_dn = "uni/vmmp-VMware/dom-domain_1"
   annotation = "annotation"
-  binding_type = "dynamicBinding"
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.